include/oneapi/tbb/detail/_flow_graph_node_impl.h

```diff
@@ -496,7 +496,7 @@ class multifunction_input : public function_input_base<Input, Policy, A, multifu
     // constructor
     template<typename Body>
     multifunction_input(graph &g, size_t max_concurrency,Body& body, node_priority_t a_priority )
-      : base_type(g, max_concurrency, a_priority, noexcept(body(input_type(), my_output_ports)))
+      : base_type(g, max_concurrency, a_priority, noexcept(tbb::detail::invoke(body, input_type(), my_output_ports)))
```

Comment:
I suppose you intend to exclude the use of invoke in the apply_body_impl_bypass. Is there a chance that the input message could define a function that would accept the output ports, and so be a valid use case?

Reply:
I have excluded `apply_body_impl_bypass` because we call operator() of `multifunction_body_leaf` there and it is always a callable object defined inside oneTBB. It supports `std::invoke` inside, so use-case you described would work and we have a test for it.